docs/src/almalinux_instructions.rst

```diff
@@ -0,0 +1,257 @@
+.. _almalinux_instructions:
+
+Step-by-step instructions for Almalinux 8.4
+===========================================
+
+.. note::
+
+    These instructions should also work for RHEL8, CentOS8, and Rocky Linux. Some changes may be necessary for RHEL7/CenOS7.
+
+
+1. Install Node.js
+~~~~~~~~~~~~~~~~~~
+
+carta-controller uses `Node.js <https://nodejs.org/>`_ and supports v12, v14, and v16. Node.js can easily be installed from the AlmaLinux AppStream repository. Here we install v14, as well as the ``npm`` package manager.
```

Comment:
"carta-controller" should be replaced with "the CARTA controller" here and in other places in this document.